cmd/leanix-k8s-connector/main.go

```diff
@@ -65,10 +65,14 @@ func main() {
 	if err != nil {
 		log.Error(err)
 	}
-	log.Info("Uploading connector logs to iHub")
-	err = storage.UploadConnectorLog(startResponse.ConnectorLoggingUrl, debugLogBuffer.Bytes())
-	if err != nil {
-		log.Error(err)
+	if startResponse.ConnectorLoggingUrl != "" {
+		log.Info("Uploading connector logs to iHub")
+		err = storage.UploadConnectorLog(startResponse.ConnectorLoggingUrl, debugLogBuffer.Bytes())
+		if err != nil {
+			log.Error(err)
+		}
+	} else {
+		log.Error("Uploading connector logs to iHub failed as 'connectorLoggingUrl' is not available.")
 	}
```

Comment:
In golang we're trying to fail fast and avoid else cases.

Here you could check if start.Response.ConnectorLoggingUrl is empty within the UploadConnectorLog function and return that as an error to the main.go